x-pack/plugins/security_solution/server/fleet_integration/fleet_integration.test.ts

```diff
@@ -61,6 +62,10 @@ describe('ingest_integration tests ', () => {
     licenseEmitter = new Subject();
     licenseService = new LicenseService();
     licenseService.start(licenseEmitter);
+
+    jest
+      .spyOn(endpointAppContextMock.endpointMetadataService, 'getFleetEndpointPackagePolicy')
```

Comment:
do you now if `spyOn()` is reset on each test iteration like other mocks?

Reply:
good question. it just creates a regular `jest.fn()` mock, but I can't find any jest configuration for auto clear/rest mocks in kibana? I added a `jest.clearAllMocks()` to the aftereach but if you know for sure we're configured for auto clear, I'll take it off.